### PR TITLE
Update to sample config and docs. Closes #49.

### LINKED ISSUE #49: Doc Bug: Some Sample Configuration Missing from Documentation
The file `sample.env` contains the following env vars which are not then referenced in any of the documentation:

- `BASENAME`
- `LOCATION`
- `OTEL_METRIC_EXPORT_INTERVAL`
- `OTEL_SERVICE_NAME`

(this refers to code within `stuart/limits-update` branch)

### PR DESCRIPTION
**Pull request by martinpeck**
Updated and tidied up the `sample.env` removing some of the config in this file where the values were already the defaults.

Updated the docs to refer to the two supported Open Telemetry config values.

### CODE REVIEW

File: sample.env [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
+# Create a copy of this file called .env, and ensure that all variables have valid values.
 
-# Add values to configure the simulator (see README.md)
-SIMULATOR_MODE=generate
+# Deployment Config
+BASENAME=__CHANGE_ME___                                                         # Prefix for assets deployed into Azure
+LOCATION=__CHANGE_ME___                                                         # Azure region the assets will be deployed to 
 
-# Set this to False to return 404 for deployments that aren't configured
-ALLOW_UNDEFINED_OPENAI_DEPLOYMENTS=True
```
Comment (stuartleeks) [inline sample.env:9]: Was this value intentionally removed?

**Review by stuartleeks — APPROVED** [review]
Thanks - hopefully the groupings will make it easier to work with the `.env` file 😄

File: sample.env (hunk 1 above)
Comment (martinpeck) [reply sample.env:9]: Yeah. The default is True, so I didn't see why this would be needed in the sample config.

Comment (martinpeck) [reply sample.env:9]: Are you OK with me removing this, given the default is already True?

Comment (stuartleeks) [reply sample.env:9]: Yeah, let's leave it as-is 👍

**Pull request merged**